core/src/main/scala/cilib/Eval.scala

```diff
@@ -16,7 +16,7 @@ sealed abstract class Eval[F[_],A] {
   def eval: RVar[NonEmptyList[A] => Objective[A]] =
     RVar.point { (fa: NonEmptyList[A]) =>
       this match {
-        case Unconstrained(f, _)   => Single(Feasible(f(F.toInput(fa))), List.empty)
+        case Unconstrained(f, _) => Single(Feasible(f(F.toInput(fa))), List.empty)
```

Comment:
It looks like all the other pattern matches are aligned on `=>` except this one

Reply:
It seems to only line them up if all the case patterns have values to the right.Only unconstraint branch does